hugegraph-store/hg-store-grpc/src/main/dev/org/apache/hugegraph/store/grpc/stream/store_stream.proto

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+import "store_common.proto";
+import "store_stream_meta.proto";
+option java_multiple_files = true;
+option java_package = "org.apache.hugegraph.store.grpc.stream";
+option java_outer_classname = "HgStoreStreamProto";
+
+/**
+   In order to improve performance, reuse memory, and reduce gc recycling, the KvStream.writeTo method needs to be overwrite.
+ */
+service HgStoreStream {
+
+  rpc Scan(stream ScanStreamReq) returns (stream KvPageRes) {}
+  rpc ScanOneShot(ScanStreamReq) returns (KvPageRes) {}
+  rpc ScanBatch(stream ScanStreamBatchReq) returns (stream KvPageRes) {}
+  rpc ScanBatch2(stream ScanStreamBatchReq) returns (stream KvStream) {}
```

Comment:
These two methods are expected to return different results, one is `KvPageRes`, the other is `KvStream`, and they are both used in `org.apache.hugegraph.store.client.NodeTxSessionProxy`.

However, these two methods indeed seem to be confused, since their similar and casual names. In subsequent migration work, we will consider renaming or refactoring it.